python/pyspark/ml/tuning.py

```diff
@@ -33,6 +33,8 @@
 
 class ParamGridBuilder(object):
     r"""
+    .. note:: Experimental
```

Comment:
I don't think it's required there? Anyway I left it as is
On Tue, 10 May 2016 at 23:21, Holden Karau <notifications@github.com> wrote:

> In python/pyspark/ml/tuning.py
> <https://github.com/apache/spark/pull/12967#discussion_r62754661>:
>
> > @@ -33,6 +33,8 @@
> >
> >  class ParamGridBuilder(object):
> >      r"""
> > +    .. note:: Experimental
>
> r is for raw string literal (often used to avoid needing certain things)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12967/files/3e9fded2a20ecbbcfa51324eaac1e06d4a19f963#r62754661>
>